### PR TITLE
[1.0] Backport of #3928

### PR DESCRIPTION
**Pull request by nabobalis**
Backport of #3928

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -168,6 +145,7 @@ workflows:
   figure-tests:
     jobs:
       - figure-tests-36
+      - figure-tests-37-astropydev
```
Comment (nabobalis) [inline .circleci/config.yml]:
```suggestion
```

Comment (Cadair) [reply .circleci/config.yml]: for why?

Comment (nabobalis) [reply .circleci/config.yml]: ?

Comment (Cadair) [reply .circleci/config.yml]: oh did this job just not work on 1.0?

Comment (nabobalis) [reply .circleci/config.yml]: Yeah, we never backported this to 1.0

**Pull request merged**